src/UI/TransactionsToastList/components/TransactionToast/components/CloseButton/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+interface CloseButtonType {
+  onDelete: () => void;
+  style: any;
+  isPending: boolean;
+}
+
+const CloseButton = (props: CloseButtonType) => {
+  const { onDelete, style, isPending } = props;
+
+  if (isPending) {
+    return null;
+  }
+
+  return (
+    <button type='button' className={style.close} onClick={onDelete}>
```

Comment:
I don't think it's a good idea to pass the whole style object only for reading the close class